### PR TITLE
F 145/update oneke from 1.29 to 1.31

### PR DESCRIPTION
**Pull request by aleixrm**
new OneKE1.31 version appliance yamls

### CODE REVIEW

File: appliances/OneKE_1.31/2a902664-98ed-4e78-80b6-af97dfd75371.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+name: OneKE 1.31
+version: 1.31.2-6.10.1-1-20241205
```
Comment (aleixrm) [inline appliances/OneKE_1.31/2a902664-98ed-4e78-80b6-af97dfd75371.yaml:3]: TODO: Update

File: appliances/OneKE_1.31/3f8627c5-9e05-4821-8ef1-20533ad9b7ba.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+name: OneKE 1.31 OS disk
+version: 1.31.2-6.10.1-1-20241205
```
Comment (aleixrm) [inline appliances/OneKE_1.31/3f8627c5-9e05-4821-8ef1-20533ad9b7ba.yaml:3]: TODO: Update

File: appliances/OneKE_1.31/3f8627c5-9e05-4821-8ef1-20533ad9b7ba.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- service
+format: qcow2
+creation_time: 1715358765
+os-id: none
+os-arch: all
+hypervisor: ALL
+opennebula_version: 6.2, 6.3, 6.4, 6.6, 6.8, 6.10
+images:
+- name: oneke_os
+  url: http://services:9999/images/service_OneKE.qcow2
```
Comment (aleixrm) [inline appliances/OneKE_1.31/3f8627c5-9e05-4821-8ef1-20533ad9b7ba.yaml:22]: TODO: Update

File: appliances/OneKE_1.31/3f8627c5-9e05-4821-8ef1-20533ad9b7ba.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  type: OS
+  dev_prefix: vd
+  driver: qcow2
+  size: 6843545600
+  checksum:
+    md5: 930437e8efaa66462ed8a3e0ebf5d693
+    sha256: d8975de1a60ce13c74e04c41d3d5142eaecd58b13df1a633c27461ea401f0aff
```
Comment (aleixrm) [inline appliances/OneKE_1.31/3f8627c5-9e05-4821-8ef1-20533ad9b7ba.yaml:29]: TODO: Update both md5 and sha256

File: appliances/OneKE_1.31/4bea23af-09c2-4520-8f3a-ce902eb5157c.yaml [hunk 5]
```diff
@@ -0,0 +1,76 @@
+---
+name: OneKE 1.31 VNF
+version: 1.31.2-6.10.1-1-20241205
```
Comment (aleixrm) [inline appliances/OneKE_1.31/4bea23af-09c2-4520-8f3a-ce902eb5157c.yaml:3]: TODO: Update

File: appliances/OneKE_1.31/4bea23af-09c2-4520-8f3a-ce902eb5157c.yaml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  nic_default:
+    model: virtio
+  lxd_security_privileged: 'true'
+  os:
+    arch: x86_64
+  user_inputs: {}
+logo: router.png
+images:
+- name: oneke_vnf
+  url: http://services/images/service_VRouter.qcow2
```
Comment (aleixrm) [inline appliances/OneKE_1.31/4bea23af-09c2-4520-8f3a-ce902eb5157c.yaml:69]: TODO: Update

File: appliances/OneKE_1.31/4bea23af-09c2-4520-8f3a-ce902eb5157c.yaml [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+  type: OS
+  dev_prefix: vd
+  driver: qcow2
+  size: 2147483648
+  checksum:
+    md5: b97e2488992abf7d633d94c3043b1888
+    sha256: 1ac6f8310da672a17b0a0a530041a40da952527be11c94937225045195834293
```
Comment (aleixrm) [inline appliances/OneKE_1.31/4bea23af-09c2-4520-8f3a-ce902eb5157c.yaml:76]: TODO: Update both md5 and sha256

File: appliances/OneKE_1.31/98df67fd-6dbd-49e3-a99d-9b8d8519c418.yaml [hunk 8]
```diff
@@ -0,0 +1,64 @@
+---
+name: OneKE 1.31 Storage
+version: 1.31.2-6.10.1-1-20241205
```
Comment (aleixrm) [inline appliances/OneKE_1.31/98df67fd-6dbd-49e3-a99d-9b8d8519c418.yaml:3]: TODO: Update

File: appliances/OneKE_1.31/e73d2dda-1816-4f47-981f-72f1255f94ac.yaml [hunk 9]
```diff
@@ -0,0 +1,227 @@
+---
+name: Service OneKE 1.31
+version: 1.31.2-6.10.1-1-20241205
```
Comment (aleixrm) [inline appliances/OneKE_1.31/e73d2dda-1816-4f47-981f-72f1255f94ac.yaml:3]: TODO: Update

File: appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml [hunk 10]
```diff
@@ -0,0 +1,29 @@
+---
+name: OneKE 1.31 Storage disk
+version: 1.31.2-6.10.1-1-20241205
```
Comment (aleixrm) [inline appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml:3]: TODO: Update

File: appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+images:
+- name: oneke_storage
+  url: https://d24fmfybwxpuhu.cloudfront.net/service_OneKE_storage-6.10.0-1-20240508.qcow2 #TODO: Replace it by the new version
+  type: OS
+  dev_prefix: vd
+  driver: qcow2
+  size: 10737418240
+  checksum:
+    md5: 1d4589798b8a63a6afa7150492ca3193
+    sha256: 23a1edeac969b498874484637169fd08de4a7f18f438594fee7103236ba000db
```
Comment (aleixrm) [inline appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml:29]: TODO: Update both md5 and sha256

File: appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+- service
+format: qcow2
+creation_time: 1715358765
+os-id: none
+os-arch: all
+hypervisor: ALL
+opennebula_version: 6.2, 6.3, 6.4, 6.6, 6.8, 6.10
+images:
+- name: oneke_storage
+  url: https://d24fmfybwxpuhu.cloudfront.net/service_OneKE_storage-6.10.0-1-20240508.qcow2 #TODO: Replace it by the new version
```
Comment (aleixrm) [inline appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml:22]: TODO: Update

File: appliances/OneKE_1.31a/88fbf221-418f-40cb-bde7-354716fe6512.yaml [hunk 13]
```diff
@@ -0,0 +1,71 @@
+---
+name: OneKE 1.31 Airgapped
+version: 1.31.4-6.10.0-1-20240510 #TODO 1.29.4-6.10.0-1-20240510
```
Comment (aleixrm) [inline appliances/OneKE_1.31a/88fbf221-418f-40cb-bde7-354716fe6512.yaml:3]: TODO: Update

File: appliances/OneKE_1.31a/d84d44f4-0e30-412a-a6f6-b1721bf7159e.yaml [hunk 14]
```diff
@@ -0,0 +1,60 @@
+---
+name: OneKE 1.31 Airgapped Storage
+version: 1.31.4-6.10.0-1-20240510 #TODO 1.29.4-6.10.0-1-20240510
```
Comment (aleixrm) [inline appliances/OneKE_1.31a/d84d44f4-0e30-412a-a6f6-b1721bf7159e.yaml:3]: TODO: Update

File: appliances/OneKE_1.31a/d1351342-44d9-454c-b2d8-38fae62296cc.yaml [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+- name: oneke_os
+  url: >-
+    http://services:9999/images/service_OneKEa.qcow2
+  type: OS
+  dev_prefix: vd
+  driver: qcow2
+  size: 26843545600
+  checksum:
+    md5: c20b412e6da462865fd1c60212910479
+    sha256: 83017ab4cf1c38db9e7f28dfdc86395e4f60419bbdbbf389d1c33969b8a6a756
```
Comment (aleixrm) [inline appliances/OneKE_1.31a/d1351342-44d9-454c-b2d8-38fae62296cc.yaml:31]: TODO: Update both md5 and sha256

File: appliances/OneKE_1.31a/d1351342-44d9-454c-b2d8-38fae62296cc.yaml [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+format: qcow2
+creation_time: 1715358765
+os-id: none
+os-arch: all
+hypervisor: ALL
+opennebula_version: >-
+  6.2, 6.3, 6.4, 6.6, 6.8, 6.10
+images:
+- name: oneke_os
+  url: >-
```
Comment (aleixrm) [inline appliances/OneKE_1.31a/d1351342-44d9-454c-b2d8-38fae62296cc.yaml:23]: TODO: Update

File: appliances/OneKE_1.31a/d1351342-44d9-454c-b2d8-38fae62296cc.yaml [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+    http://services:9999/images/service_OneKEa.qcow2
+  type: OS
+  dev_prefix: vd
+  driver: qcow2
+  size: 26843545600
```
Comment (aleixrm) [inline appliances/OneKE_1.31a/d1351342-44d9-454c-b2d8-38fae62296cc.yaml:28]: TODO: Update

File: appliances/OneKE_1.31a/d1351342-44d9-454c-b2d8-38fae62296cc.yaml [hunk 18]
```diff
@@ -0,0 +1,31 @@
...
+  OneKE 1.31 Airgapped appliance OS disk.
+
+  See the dedicated [documentation](https://github.com/OpenNebula/one-apps/wiki).
+short_description: OneKE 1.31 Airgapped appliance OS disk
+tags:
+- disk
+- kubernetes
+- service
+format: qcow2
+creation_time: 1715358765
```
Comment (aleixrm) [inline appliances/OneKE_1.31a/d1351342-44d9-454c-b2d8-38fae62296cc.yaml:15]: TODO: Update

File: appliances/OneKE_1.31a/d84d44f4-0e30-412a-a6f6-b1721bf7159e.yaml [hunk 19]
```diff
@@ -0,0 +1,60 @@
...
+
+  See the dedicated [documentation](https://github.com/OpenNebula/one-apps/wiki).
+short_description: OneKE 1.31 Airgapped VM template for Storage VMs
+tags:
+- kubernetes
+- service
+- ubuntu
+type: VMTEMPLATE
+format: qcow2
+creation_time: 1715358765
```
Comment (aleixrm) [inline appliances/OneKE_1.31a/d84d44f4-0e30-412a-a6f6-b1721bf7159e.yaml:16]: TODO: Update

File: appliances/OneKE_1.31a/9378a80f-7556-46d2-8a89-43c8b03162e6.yaml [hunk 20]
```diff
@@ -0,0 +1,223 @@
...
+  - [OneKE 1.31 Airgapped](/appliance/88fbf221-418f-40cb-bde7-354716fe6512)
+  - [OneKE 1.31 VNF](/appliance/4bea23af-09c2-4520-8f3a-ce902eb5157c)
+  - [OneKE 1.31 Airgapped Storage](/appliance/d84d44f4-0e30-412a-a6f6-b1721bf7159e)
+short_description: Multi-master Kubernetes 1.31 Airgapped cluster for KVM and vCenter hosts, orchestrated by OneFlow
+tags:
+- kubernetes
+- oneflow
+- service
+- ubuntu
+creation_time: 1715358765
```
Comment (aleixrm) [inline appliances/OneKE_1.31a/9378a80f-7556-46d2-8a89-43c8b03162e6.yaml:28]: TODO: Update

File: appliances/OneKE_1.31a/88fbf221-418f-40cb-bde7-354716fe6512.yaml [hunk 21]
```diff
@@ -0,0 +1,71 @@
...
+
+  See the dedicated [documentation](https://github.com/OpenNebula/one-apps/wiki).
+short_description: OneKE 1.31 Airgapped VM template for Master and Worker VMs
+tags:
+- kubernetes
+- service
+- ubuntu
+type: VMTEMPLATE
+format: qcow2
+creation_time: 1715358765
```
Comment (aleixrm) [inline appliances/OneKE_1.31a/88fbf221-418f-40cb-bde7-354716fe6512.yaml:16]: TODO: Update

File: appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml [hunk 22]
```diff
@@ -0,0 +1,29 @@
...
+  OneKE 1.31 appliance Storage disk (empty).
+
+  See the dedicated [documentation](https://github.com/OpenNebula/one-apps/wiki).
+short_description: OneKE 1.31 appliance Storage disk
+tags:
+- disk
+- kubernetes
+- service
+format: qcow2
+creation_time: 1715358765
```
Comment (aleixrm) [inline appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml:15]: TODO: Update

File: appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml [hunk 23]
```diff
@@ -0,0 +1,29 @@
...
+os-arch: all
+hypervisor: ALL
+opennebula_version: 6.2, 6.3, 6.4, 6.6, 6.8, 6.10
+images:
+- name: oneke_storage
+  url: https://d24fmfybwxpuhu.cloudfront.net/service_OneKE_storage-6.10.0-1-20240508.qcow2 #TODO: Replace it by the new version
+  type: OS
+  dev_prefix: vd
+  driver: qcow2
+  size: 10737418240
```
Comment (aleixrm) [inline appliances/OneKE_1.31/eb2558c9-9be8-4eff-8b41-c2e35a0d4e74.yaml:26]: TODO: Update

File: appliances/OneKE_1.31/e73d2dda-1816-4f47-981f-72f1255f94ac.yaml [hunk 24]
```diff
@@ -0,0 +1,227 @@
...
+  - [OneKE 1.31 VNF](/appliance/4bea23af-09c2-4520-8f3a-ce902eb5157c)
+  - [OneKE 1.31 Storage](/appliance/98df67fd-6dbd-49e3-a99d-9b8d8519c418)
+short_description: Multi-master Kubernetes 1.31 cluster for KVM hosts, orchestrated
+  by OneFlow
+tags:
+- kubernetes
+- oneflow
+- service
+- ubuntu
+creation_time: 1715358765
```
Comment (aleixrm) [inline appliances/OneKE_1.31/e73d2dda-1816-4f47-981f-72f1255f94ac.yaml:29]: TODO: Update

File: appliances/OneKE_1.31/98df67fd-6dbd-49e3-a99d-9b8d8519c418.yaml [hunk 25]
```diff
@@ -0,0 +1,64 @@
...
+
+  See the dedicated [documentation](https://github.com/OpenNebula/one-apps/wiki).
+short_description: OneKE 1.31 VM template for Storage VMs
+tags:
+- kubernetes
+- service
+- ubuntu
+type: VMTEMPLATE
+format: qcow2
+creation_time: 1715358765
```
Comment (aleixrm) [inline appliances/OneKE_1.31/98df67fd-6dbd-49e3-a99d-9b8d8519c418.yaml:16]: TODO: Update

File: appliances/OneKE_1.31/4bea23af-09c2-4520-8f3a-ce902eb5157c.yaml [hunk 26]
```diff
@@ -0,0 +1,76 @@
...
+
+  See the dedicated [documentation](https://github.com/OpenNebula/one-apps/wiki).
+short_description: Appliance with several Virtual Network Functions to run as VM
+tags:
+- alpine
+- kubernetes
+- service
+- vnf
+format: qcow2
+creation_time: 1715784892
```
Comment (aleixrm) [inline appliances/OneKE_1.31/4bea23af-09c2-4520-8f3a-ce902eb5157c.yaml:17]: TODO: Update

File: appliances/OneKE_1.31/3f8627c5-9e05-4821-8ef1-20533ad9b7ba.yaml [hunk 27]
```diff
@@ -0,0 +1,29 @@
...
+os-arch: all
+hypervisor: ALL
+opennebula_version: 6.2, 6.3, 6.4, 6.6, 6.8, 6.10
+images:
+- name: oneke_os
+  url: http://services:9999/images/service_OneKE.qcow2
+  type: OS
+  dev_prefix: vd
+  driver: qcow2
+  size: 6843545600
```
Comment (aleixrm) [inline appliances/OneKE_1.31/3f8627c5-9e05-4821-8ef1-20533ad9b7ba.yaml:26]: TODO: Update

File: appliances/OneKE_1.31/2a902664-98ed-4e78-80b6-af97dfd75371.yaml [hunk 28]
```diff
@@ -0,0 +1,76 @@
...
+
+  See the dedicated [documentation](https://github.com/OpenNebula/one-apps/wiki).
+short_description: OneKE 1.31 VM template for Master and Worker VMs
+tags:
+- kubernetes
+- service
+- ubuntu
+type: VMTEMPLATE
+format: qcow2
+creation_time: 1715358765
```
Comment (aleixrm) [inline appliances/OneKE_1.31/2a902664-98ed-4e78-80b6-af97dfd75371.yaml:16]: TODO: Update

**Review by sk4zuzu — APPROVED** [review]
LGTM

**Pull request merged**